src/server/services/wfs/qgswfsgetfeature.cpp

```diff
@@ -1062,7 +1062,7 @@ namespace QgsWfs
           }
         }
         // EPSG:4326 max extent is -180, -90, 180, 90
-        rect = new QgsRectangle( rect->intersect( new QgsRectangle( -180.0, -90.0, 180.0, 90.0 ) ) );
+        rect = new QgsRectangle( rect->intersect( QgsRectangle( -180.0, -90.0, 180.0, 90.0 ) ) );
```

Comment:
Note to server team: this still leaks, because it overwrites the existing pointer value without freeing it. It just leaks half as much!